packages/trail-fastify-plugin/lib/routes/trails.js

```diff
@@ -40,9 +40,14 @@ module.exports = async function (fastify, options) {
     schema: {
       query: trailSchema.search,
       response: {
-        200: S.array()
-          .description('The search results.')
-          .items(trailSchema.response),
+        200: S.object()
```

Comment:
shouldn't this whole object be captured in a schema imported from the schemas module?

Reply:
Agree .. was being a little lazy :)

Reply:
moving to schema will not be able to reference the array of `trailschema.response`. Also I am not much conversant with the schema object and how changing it will impact the generated documentation.

Perhaps I can move the response object outside, and then reference it in the trailSchema

Reply:
because  this is the new shape of the response, trailSchema.response should change to reflect that the response is now an object with 2 properties instead of an array. makes sense?

Reply:
The trailSchema.response is still being used for the `POST /trails` response, hence added a new object searchResponse in the schema